README.md

```diff
@@ -38,7 +38,6 @@ Logging of this plugin has several categories:
 * `backendRoot`
 * `branchLayout`
 * `frontendActions`
-* `frontendGraph`
```

Comment:
No such tag yet? :o

Reply:
no, finally I'm not created any logging in this module